plugin/src/withReactNativeFbads.ts

```diff
@@ -0,0 +1,85 @@
+import { AndroidConfig, ConfigPlugin, createRunOncePlugin, withAndroidManifest } from "@expo/config-plugins";
+import { getMainApplicationOrThrow, prefixAndroidKeys } from "@expo/config-plugins/build/android/Manifest";
```

Comment:
Can use `AndroidConfig.Manifest.getMainApplicationOrThrow`, etc. to prevent internal imports.